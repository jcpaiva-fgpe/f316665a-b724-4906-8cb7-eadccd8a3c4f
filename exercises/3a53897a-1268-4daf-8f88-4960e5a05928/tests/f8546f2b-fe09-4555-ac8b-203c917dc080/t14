-1664 22222 -1656 22234
10
6 6
-6 -6
4 4
-8 -8
-18 18
10 -10
6 -6
4 -4
-3200 0
0 500
