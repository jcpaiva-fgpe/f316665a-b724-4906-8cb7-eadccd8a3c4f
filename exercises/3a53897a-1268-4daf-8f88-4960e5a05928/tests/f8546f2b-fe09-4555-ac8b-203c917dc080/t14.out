1
Madrid 5
